### ISSUE TITLE
deploy using AURO

### ISSUE THREAD

**Issue by marekyggdrasil** [sqrt-xx/mina-ui-deployment#1]
Related Discord discussion: https://discord.com/channels/484437221055922177/915745847692636181/1109079490111479848

Currently, if you run this repo while trying to deploy using AURO wallet you will encounter this issue

<img width="348" alt="Screenshot 2023-05-19 at 1 17 44 PM" src="https://github.com/sqrt-xx/mina-ui-deployment/assets/8202674/58e68f73-a5f5-4dfa-b0b9-fa46521153bb">

I am attempting o debug it. The message indicates signature is missing.

## Option 1 - Use AccountUpdate

Maybe I am not building the transaction correctly. Perhaps there is a way to indicate signature is needed using an AccountUpdate object and then AURO would automatically find out signature is required and produce it. Seems reasonable as it is hard for me to believe AURO can only broadcast proved transactions and not signed ones.

## Option 2 - Manually sign using AURO

I suspect that I would need to manipulate transactionJSON
https://github.com/sqrt-xx/mina-ui-deployment/blob/dd7b17ee588f8f75d2376044d233bbe81d27e14a/ui/pages/_app.page.tsx#L235

in a way similar to how its done in SnarkyJS
https://github.com/o1-labs/snarkyjs/blob/83745578ffe92efdb914abc89cd448f5f7c8a1a0/src/lib/account_update.ts#L2054-L2084

basically sign the fee and each account update, but instead of Ledger I would need to use sign fields from AURO
https://github.com/aurowallet/auro-test-dapp/blob/9c66008f2f26907ffa1d789c817c7b55b1fe5c7b/src/index.js#L186-L203

**Comment by marekyggdrasil** [comment]
Big thanks to @mitschabaude for finding the solution!

https://github.com/sqrt-xx/mina-ui-deployment/commit/8669a1d40dc5c265d518b8923f8d594e61c197a4

Discord ref: https://discord.com/channels/484437221055922177/1109079490111479848/1110669468629209219
